src/odemis/odemisd/mdupdater.py

```diff
@@ -359,6 +362,20 @@ def updatePosition(unused, analyzer=analyzer, comp_affected=comp_affected):
 
         return True
 
+    def observeStreakLens(self, streak_lens, comp_affected):
+        """Update the magnification of the streak lens affecting the
+        streak readout camera."""
+
```

Comment:
It works fine now, as long as the only component in the microscope file is the `streak-ccd`. However, to be extra safe, I'd recommend to explicitly check for it, as in `observeLens()`. Something like:
```
     if not comp.role.endswith("ccd"):
            return False
```